happi/client.py

```diff
@@ -627,7 +664,7 @@ def search_regex(
         items = self.backend.find_regex(kwargs, flags=flags)
         return self._get_search_results(items)
 
-    def export(self, path=sys.stdout, sep='\t', attrs=None):
+    def export(self, path=sys.stdout, sep='\t', attrs: Optional[list[str]] = None):
```

Comment:
Question pre-dating PR: why does this function exist? Does it have utility over the json backend?

Reply:
Follow up issue made #365